example/contract-schema.ts

```diff
@@ -3,7 +3,7 @@ import { InMemorySigner } from '@taquito/signer';
 
 
 async function example() {
-    const provider = 'https://api.tez.ie/rpc/florencenet';
+    const provider = 'https://api.tez.ie/rpc/granadanet';
```

Comment:
The example won't work because KT1SwtWpUE1TpBF8c4T1c2NsaDGgjKv3Xwy7 is on florencenet

Reply:
i will be fixing example cases in next PR